### ISSUE TITLE
a11y warnings

### ISSUE THREAD

**Issue by laoshaw** [ItalyPaleAle/svelte-spa-router#292]
when use `<a user:link{{href:"/wild/card", disabled:false}}>Wildcard route</a>` my editor complains "a11y-missing-attribute": A11y: <a> element should have an href attribute. This is not a bug just wondering what's the best approach to silence the warning in this case.

**Comment by ItalyPaleAle** [comment]
Use another tag (for example a span) with role=link

Or just disable the warning (but I would recommend the other option)

Docs: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/link_role
